client/extensions/woocommerce/state/sites/shipping-zone-locations/selectors.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * External dependencies
+ */
+import { get, isObject } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import { getSelectedSiteId } from 'state/ui/selectors';
+import { LOADING } from 'woocommerce/state/constants';
+
+const getRawShippingZoneLocations = ( state, siteId = getSelectedSiteId( state ) ) => {
```

Comment:
If this is just for local use (inside this file), not exported, I think `getRaw…` is fine. If it's something exported for general use, we'd want to have a better name.